elements/lisk-transaction-pool/src/transaction_list.ts

```diff
@@ -25,6 +25,10 @@ const DEFAULT_MAX_SIZE = 64;
 // tslint:disable-next-line no-magic-numbers
 export const DEFAULT_MINIMUM_REPLACEMENT_FEE_DIFFERENCE = BigInt(10);
 
+type AddResult =
```

Comment:
This name doesn't allow to quickly figure out what it stores; is it the result of adding two values or the result of an `Add` operation? A better name would be great.